ReplayKitExample/BroadcastExtension/ExampleReplayKitAudioCapturer.m

```diff
@@ -8,7 +8,7 @@
 #import "ExampleReplayKitAudioCapturer.h"
 
 // Our guess at the maximum slice size used by ReplayKit. We have observed 1024 in the field.
-static size_t kMaximumFramesPerBuffer = 2048;
+static size_t kMaximumFramesPerBuffer = 44256;
```

Comment:
Should we update the documentation here about why 44256?

Reply:
I agree. There isn't much here to justify why we need to handle such a large amount of samples at a time. Is delivering ~1 second chunks of audio the norm, or just a one time thing from ReplayKit?

Reply:
Let's discuss this live.

Reply:
To capture feedback from the live discussion:

1. We would like to know if the large audio samples are a one time thing, or continuous.
2. If we always receive so many frames at a time, then why? We should file a bug against ReplayKit since this introduces too much delay.
3. If we need this high limit, only spend the memory if the developer has enabled capturing app audio.